README.md

```diff
@@ -40,6 +40,14 @@ The architecture with the components involved is documented [here](docs/architec
 
 ## Deployment and examples
 
+### Generate example configurations
+
+Generates example configurations (CRs and provider components)
+
+```sh
+    make generate-examples
```

Comment:
do we need this here ? it is automatically triggered as part of `make deploy` or `make install`

Reply:
If I follow the instructions as it is now from the current flow in README, I hit a bump. because the first instruction is, `make deploy-bmo-cr`. And the CR is not there. Then I realized , i didn't generate the examples. For someone who is new, he would not know such a phony exists and if he simple starts from README, since it says deployment and examples in the heading, he will hit the issue too. In my opinion, this should be visible here.

Reply:
hum, the problem is rather that the generate-examples dependency was removed from the deploy-bmo-cr target. I don't recall why. We probably should fix it there so that it would work out of the box, or deploy directly from the examples folder, not the _out folder

Reply:
True. I have updated the PR